pom.xml

```diff
@@ -33,7 +33,7 @@
         <non.final>false</non.final>
         <extension.name>javax.security.auth.message</extension.name>
         <spec.version>1.1</spec.version>
-        <findbugs.version>2.4.0</findbugs.version>
+        <findbugs.version>3.0.1</findbugs.version>
```

Comment:
It would be good to change over to SpotBugs at some point.

Reply:
I don't know SpotBugs too well, but I'll look into it. Thx!